_posts/2024-01-11-Socratic-Seminar-50.md

```diff
@@ -72,3 +72,7 @@ Discussion points:
 * Boltz swaps taking 0-conf risks
 * btc "Savings Account"... in a hot wallet?
 * Global south hodling on Liquid?
+
+## Tidwell Game presentation
```

Comment:
We typically don't have presentations at the Socratic Seminar.    I would want to check this out before we merge this to make sure that (1) it's very short and (2) that it's general technical material suitable for BitDevs.  Is the presentation available somewhere for review?

Reply:
cc @miketwenty1

Reply:
I'm definitely happy to present and do a little 2 minute demo of a bitcoin game i'm working on and take questions.. but it's totally not necessary! I think rockstar is throwing me a bone here, 🙏 thank you.

We can avoid entirely if it's not kosher for the meet. 
I will also be making a PR later today to suggest a topic.

Reply:
Ok, I see.  Let's proceed as usual then, sticking to discussion of topics for the Socratic Seminar.  Then afterward, you can use the projector and demo to anyone who is interested.

Reply:
great, look forward to it. I submitted a PR, if time remains after I can shill my stuff haha.